app/mixins/scheduled_offers.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+  scheduleldOffers: true,
```

Comment:
@swatijadhav small typo: `scheduleldOffers` should be `scheduledOffers` and in a couple of other places too.